### ISSUE TITLE
AttributeError: 'Controller' object has no attribute 'revData'

### ISSUE THREAD

**Issue by jimboca** [Einstein42/udi-ecobee-poly#36]
```
2019-01-23 17:11:37,438 [Controller] [ERROR] Exception in thread Controller:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.7/threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "/app/template/pgc_interface/pgc_interface.py", line 713, in _parseInput
    self.longPoll()
  File "./ecobee-poly.py", line 230, in longPoll
    self.updateThermostats()
  File "./ecobee-poly.py", line 248, in updateThermostats
    if self.checkRev(thermostat):
  File "./ecobee-poly.py", line 263, in checkRev
    if tstat['thermostatId'] in self.revData:
AttributeError: 'Controller' object has no attribute 'revData'
```

**Comment by jimboca** [comment]
@Einstein42 Do you happen to have more of the log before this? I think I know how this could happen but not positive

**Comment by jimboca** [comment]
Added another fix to 2.0.34, hopefully that's it.
